src/components/index.tsx

```diff
@@ -6,10 +6,15 @@
 export { NavItems } from './Navbar/NavItems';
 export { Navbar } from './Navbar/Navbar';
 export { Menu } from './Navbar/Menu';
-
+/*
+ *
+ * @section About
+ *
+ */
+export { About } from './sections/About.section';
 /*
  *
  * @section FooterSection
```

Comment:
```suggestion
/*
 *
 * @section
 *
 */
export { About } from './sections/About.section'
export { FooterSection } from './sections/Footer.section';
```

Help me keep it like that, just being tiny bit picky hhh :D

Reply:
Haha all good :D I wasn't sure how we were keeping it. Resolved! :)